docs/stdlib/doc.md

```diff
@@ -0,0 +1,50 @@
+## Builtin VS StdLib
+
+[Builtins](/advanced_syntax/builtins) are method that are included in the Amber compiler and don't need to be imported in the code.
+
+In contrast, the standard library (stdlib) is a collection of Amber functions that is embedded in every Amber release. Each version of Amber may include changes to the standard library, and you need to import these functions in your code. These functions are more advanced and can accept various parameters.
+
+## Standard library advantages
+
+There are 3 reasons to use this library:
+
+* Generate valid [Shellcheck](https://www.shellcheck.net/) code (though full support for this in Amber is still in progress)
+* They have unit tests for every function
+* Extended the native Bash commands with more features
```

Comment:
I don't know if we need this section at all

Reply:
In my opinion yes, we had already people asking that different between builtins and explain it in the documentation is important.

Reply:
But it's like... basic programming knowledge and it's not really explaining anything. Builtins do these 3 things too so what is the advantage over them? It all just looks silly to me. Can't we just say that it's a big collection of usefull standardized Amber functions?

Reply:
Can you show me an example of other languages pointing to advantages of using std? To me they are pretty clear - you don't have to write them yourself. They are given to you by the creators of the language so you can feel safe using them. Shellcheck is something we can say but not make it the whole new section

Reply:
I agree that it's good to mention that standard library functions are shell-check compliant. But I don't think that we need to say that the functions are tested and that they add more value to bash. It's something that users already expect from this language

Reply:
How about this change?

```suggestion
## Standard library and Shellcheck
Just like the Amber's compiled Bash output, all standard library functions are built from the ground up to be shell check compliant. This means that you can focus more on building the logic and spend less time on keeping the code predictable and valid.
```